go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/openshift-kni/numaresources-operator
 go 1.17
 
 require (
+	github.com/fsnotify/fsnotify v1.4.9
```

Comment:
why do we need this change?

Reply:
I'm not sure, need to check

Reply:
If not obviously needed, please remove to keep the PR minimal.

Reply:
It was already required, please see some lines below.
Why it becomes from an indirect to a direct one, maybe someone forgot to run ```go mod``` but it wouldn't affect the compilation since only the ```//indirect``` comment was removed.

Reply:
sure, and this change doesn't belong to this PR. So it should be moved on a separate PR and/or expunged by this one.

Reply:
please remove this commit from the series